apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -395,6 +413,14 @@ if [[ "$should_use_xcodebuild" == true ]]; then
     args+=(-resultBundlePath "$result_bundle_path")
   fi
 
+  if [[ "$create_xctestrun_bundle" == true ]]; then
+    echo "note: creating xctestrun bundle"
+    cp "$test_tmp_dir/tests.xctestrun" "$TEST_UNDECLARED_OUTPUTS_DIR"
+    cp -R "$test_tmp_dir_test_host_path" "$TEST_UNDECLARED_OUTPUTS_DIR"
+    cp -R "$test_tmp_dir/$runner_app" "$TEST_UNDECLARED_OUTPUTS_DIR"
+    exit 0
```

Comment:
IMO, this make the implementation quite confusing since you'd run `bazel test` or `bazel coverage` but not actually run any test.

Is it possible to make this it's own rule/implementation and modularize out the functionality that creates the XCTestRun within this script to use in both places?

Reply:
Agreed, lets do this instead

Reply:
You are imagining that this would be engaged via a new rule with bazel build?

Reply:
If possible, that'd be awesome.